ltj/settings.py

```diff
@@ -123,41 +108,8 @@
 
 USE_TZ = True
 
+STATIC_URL = env('STATIC_URL')
+
+# Custom settings
 
-# Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/1.9/howto/static-files/
-
-STATIC_URL = '/static/'
-
-# Own settings
-SRID = 3879
-
-# local_settings.py can be used to override environment-specific settings
-# like database and email that differ between development and production.
-f = os.path.join(BASE_DIR, "local_settings.py")
-if os.path.exists(f):
-    import sys
-    import imp
-    module_name = "%s.local_settings" % ROOT_URLCONF.split('.')[0]
-    module = imp.new_module(module_name)
-    module.__file__ = f
-    sys.modules[module_name] = module
-    exec(open(f, "rb").read())
-
-if 'SECRET_KEY' not in locals():
-    secret_file = os.path.join(BASE_DIR, '.django_secret')
-    try:
-        SECRET_KEY = open(secret_file).read().strip()
-    except IOError:
-        import random
-        system_random = random.SystemRandom()
-        try:
-            SECRET_KEY = ''.join([system_random.choice('abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)')
-                                  for i in range(64)])
-            secret = open(secret_file, 'w')
-            import os
-            os.chmod(secret_file, 0o0600)
-            secret.write(SECRET_KEY)
-            secret.close()
-        except IOError:
-            Exception('Please create a %s file with random characters to generate your secret key!' % secret_file)
+SRID = 3879  # Spatial reference system identifier used for geometry fields
```

Comment:
Is this used somewhere already?

Reply:
yes, this used on geometry fields: https://github.com/City-of-Helsinki/ltj/blob/master/nature/models.py#L194

Reply:
I don't know how I missed that, sorry, a bad search from my side 😄 👍